examples/thresholds/README.md

```diff
@@ -0,0 +1,111 @@
+This example shows you how to run Infracost with Atlantis to post a comment on your PR when cloud costs exceed set thresholds.
+
+## Running with GitHub
+
+1. Update your setup to use the [infracost-atlantis](https://hub.docker.com/r/infracost/infracost-atlantis) dockerhub image
+2. You'll need to pass the following custom env vars into the container
+   ```sh
+   GITHUB_TOKEN=<your-github-token>
+   INFRACOST_API_KEY=<your-infracost-api-token>
+   ```
+3. Add the following yaml spec to `repos.yaml` or `atlantis.yaml` config files:
+   ```yaml
+   repos:
+     - id: /.*/
+       workflow: terraform-infracost
+   workflows:
+     terraform-infracost:
+       plan:
+         steps:
+           - init
+           - plan
+           # Run Infracost breakdown and save to a tempfile, namespaced by this project, PR, workspace and dir
+           - run: infracost breakdown --path=$PLANFILE --format=json --log-level=info --out-file=/tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json
+           - run: |
+               # Read the breakdown JSON and get costs
+               past=$(cat /tmp/$BASE_REPO_OWNER-$BASE_REPO_NAME-$PULL_NUM-$WORKSPACE-$REPO_REL_DIR-infracost.json | jq -r "(.pastTotalMonthlyCost // 0) | tonumber")
```

Comment:
This examples makes me wonder if there's any way we can make this file path shorter? Unfortunately we can't use `$PROJECT_NAME` because that's not always set. Is it possible to set an env var to that value and use that?

I think this is outside of the scope of this PR though so happy to look at it separately.

Reply:
This will work with the infracost-atlantis docker image, but not with the "Install in Docker" option (https://github.com/infracost/infracost-atlantis/tree/hail_atlantis#option-2-install-in-docker) since `jq` isn't included in the Atlantis docker image. Maybe all we need for this is a note at the top to show how to install `jq` if it's not already installed.

Reply:
> This examples makes me wonder if there's any way we can make this file path shorter? Unfortunately we can't use `$PROJECT_NAME` because that's not always set. Is it possible to set an env var to that value and use that?

yeah good idea at setting an env var - since this affects most other examples, let merge this and resolve all others

Reply:
> This will work with the infracost-atlantis docker image, but not with the "Install in Docker" option (https://github.com/infracost/infracost-atlantis/tree/hail_atlantis#option-2-install-in-docker) since `jq` isn't included in the Atlantis docker image. Maybe all we need for this is a note at the top to show how to install `jq` if it's not already installed.

I'll add a note